app/templates/main/volunteerProfile.html

```diff
@@ -129,10 +129,12 @@ <h2>Which programs do you want to make {{volunteer.firstName}} {{volunteer.lastN
                   {% set action = "remove" %}
                   {% set checked = "checked" %}
             {% endif %}
+            <div class="table-responsive">
               <table>
                     <input  type="checkbox" id="mgr-{{ program.id }}" data-programid="{{program.id}}" {{checked}}  onclick="updateManagers(this,'{{volunteer.username}}')">
                     <label  style="padding-left: 15px;" for="mgr-{{ program.id }}">{{program.programName}}</label>
               </table>
+            </div>
```

Comment:
Rather than wrap this with table-responsive we should get rid of the table. It's weird, anyway. Also, there was already an empty div wrapping this (for some reason)

Reply:
resolved